test/e2e/suite_test.go

```diff
@@ -118,7 +118,7 @@ var _ = BeforeSuite(func(done Done) {
 	k8sClient = k8sManager.GetClient()
 	Expect(k8sClient).NotTo(BeNil())
 	close(done)
-}, 60)
+}, 2000)
```

Comment:
Why it is so high?

Reply:
All tests took 1564 s. I can try different values of waiting times in all tests and in probes to make it shorter, but some tests fail, that's why I had to increase times in probes of the main jenkins creating function (not the one from the backup and restore).